wd_demo.au3

```diff
@@ -1137,6 +1137,12 @@ Func SetupGecko($bHeadless)
 	_WD_CapabilitiesAdd('alwaysMatch', 'firefox')
 	_WD_CapabilitiesAdd('browserName', 'firefox')
 	_WD_CapabilitiesAdd('acceptInsecureCerts', True)
+	; REMARK
+	; using 32bit geckodriver.exe sometimes may require to set 'binary' capabilities,
```

Comment:
`set 'binary' capabilities` 
or
`set 'binary' capability`

?

Reply:
I would recommend `set 'binary' option`

Reply:
I see that you switched to `option` and then reverted to `capabilities`. Was this intentional? I ask because there are a few other minor tweaks that I intend to make to the wording.

Reply:
Feel free to re wording any comments headers.

Reply:
Done. Can we marked this conversation as resolved?

Reply:
Yes